gauntlet/packages/gauntlet-serum-multisig/src/lib/contracts.ts

```diff
@@ -0,0 +1,45 @@
+import { io } from '@chainlink/gauntlet-core/dist/utils'
+import { PublicKey } from '@solana/web3.js'
+import { join } from 'path'
+
+export type Contract = {
+  id: CONTRACT_LIST
+  version: string
+  idl: any
+  programId: PublicKey
+}
+
+export enum CONTRACT_LIST {
+  ACCESS_CONTROLLER = 'access_controller',
+  OCR_2 = 'ocr2',
+  FLAGS = 'flags',
+  STORE = 'store',
+  TOKEN = 'token',
+  MULTISIG = 'serum_multisig',
+}
+
+export const CONTRACT_ENV_NAMES = {
+  [CONTRACT_LIST.ACCESS_CONTROLLER]: 'PROGRAM_ID_ACCESS_CONTROLLER',
+  [CONTRACT_LIST.OCR_2]: 'PROGRAM_ID_OCR2',
+  [CONTRACT_LIST.STORE]: 'PROGRAM_ID_STORE',
+  [CONTRACT_LIST.MULTISIG]: 'PROGRAM_ID_MULTISIG',
+}
+
+export const getContract = (name: CONTRACT_LIST, version: string): Contract => ({
```

Comment:
We need to avoid copy-pasting code. As importing all these utils from `gauntlet-solana-contracts` will create a circular dependency, we should bring this over `gauntlet-solana`.